src/classes/LD_LeadConvertOverride_TEST.cls

```diff
@@ -564,79 +564,84 @@ private class LD_LeadConvertOverride_TEST {
         newNativeLeadConversion_Test(CAO_Constants.BUCKET_PROCESSOR);
     }
     /*******************************************************************************************************
-    * @description Runs Native Lead Conversion with a new Contact selection for the conversion
+    * @description Runs Native Lead Conversion with an existing Contact selection for the conversion
     */
-    public static void newNativeLeadConversion_Test(string strProcessor) {
+    public static void newNativeLeadConversion_Test(String strProcessor) {
         npe01__Contacts_and_Orgs_Settings__c contactSettingsForTests = UTIL_CustomSettingsFacade.getContactsSettingsForTests(
             new npe01__Contacts_and_Orgs_Settings__c (npe01__Account_Processor__c = strProcessor));
         //CAO_Constants.setIndividualAccountForTests(CAO_Constants.INDIVIDUAL_ACCOUNT_NAME_FOR_TESTS);
 
         Contact con = new Contact();
         con.Firstname = 'Test';
-        con.Lastname = 'TestContact';
+        con.Lastname = 'NativeLead';
         //con.MailingStreet = '123';
         insert con;
-        system.assertEquals(1, [select count() from Contact]);
-        system.assertEquals(1, [select count() from Account]);
+
+        System.assertEquals(1, [SELECT count() FROM Contact]);
```

Comment:
Not something we necessarily need to take action on, but I wonder if these extraneous asserts are really necessary.  I can see how they might help troubleshoot in the case of test data issues, but otherwise they seem just like filler code without much nutritional value.  Just a thought.